content/blog/2022-03-05_newsletter.md

```diff
@@ -0,0 +1,185 @@
+---
+title: The XMPP Newsletter February 2022
+author: emus
+date: 2022-03-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter, great to have you here again! This issue covers the month of February 2022.
+
+Like this newsletter, many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider saying thanks or help these projects! Interested in supporting the Newsletter team? Read more at the bottom.
+
+### Newsletter translations
+
+This is a community effort, and we would like to thank translators for their contributions. Volunteers are welcome! Translations of the XMPP Newsletter will be released here (with some delay):
+
+- French: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- German: [xmpp.org/de/blog](https://xmpp.org/de/blog/) and [anoxinon.de](https://anoxinon.de/blog/)
+- Italian: [nicfab.it](https://www.nicfab.it/)
+- Spanish: [xmpp.org/es/blog](https://xmpp.org/es/blog/)
+
+## XSF Announcements
+
+![XSF and Google Summer of Code 2022](/images/logos/GSoC_2022_Logo.png "XSF and Google Summer of Code 2022")
+
+- The XSF is planning to participate in [Google Summer of Code 2022 (GSoC)](https://xmpp.org/community/gsoc-2022/). If you are interested in participating as a student, mentor or as a project in general, please [add your ideas and reach out to us](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022).
+- All XSF members are called to vote in the currently on-going member applications.
+
+## XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+## Events
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Videos
+
+
+## Articles
+
+- JMP.chat [describes in their blog](http://blog.jmp.chat/b/2022-bidirectional-gateways) why gateways, and especially bidirectional gateways) with XMPP and other protocols, are so useful. [Their Newsletter](http://blog.jmp.chat/b/february-newsletter-2022) also announces the launch of worldwide calling and completion of their light rebrand.
```

Comment:
I'd scratch "light" and just call it a rebrand. It's just my personal preference to remove unnecessary adjectives.

Reply:
Well, I would not treat this as unnecessary adjective, but I removed it.